docker-compose.yml

```
@@ -71,7 +71,7 @@ services:
     ports:
       - 4001:4001
       - 8080:8080
-      - 5001:5001
+      - 5002:5002
```

```suggestion
      - 5002:5001
```
As the container itself will still be listening on 5001, however on our localhost binding we want it to listen on 5002.